src/main/java/org/primefaces/component/inplace/InplaceRenderer.java

```diff
@@ -112,7 +113,7 @@ protected String getLabelToRender(FacesContext context, Inplace inplace) {
         String label = inplace.getLabel();
         String emptyLabel = inplace.getEmptyLabel();
 
-        if (label != null) {
+        if (label != null && !isValueBlank(label)) {
```

Comment:
Write `!isValueBlank(label)` should be enough

Reply:
Thanks!  Fixed..